jvcl/locale/de/LC_MESSAGES/jvcl.po

```diff
@@ -1203,6 +1200,12 @@ msgid ""
 "2 if you want it in the second position\n"
 "...."
 msgstr ""
+"Hier die Position des Start Tags definieren\n"
+"\n"
+"1 wenn es egal ist\n"
+"0 wenn er nicht im string sein kann\n"
```

Comment:
Even if "string" is an internal type, here it should be uppercase as it's usesld as a name here.